ajax/libs/blueprintjs-core/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "blueprintjs-core",
+  "filename": "blueprint.min.css",
```

```suggestion
  "filename": "core.bundle.js",
```

This file does not appear to exist in the most recent version:

```
node_modules/@blueprintjs/core/dist
└── core.bundle.js
```